src/main/java/uk/ac/cam/cl/dtg/segue/api/monitors/SegueLoginbyIPMisuseHandler.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2015 Stephen Cummins
```

Comment:
I didn't realise you've changed your name, what's wrong with Robert? :-P

On a more serious note: do we need this type of license documentation? If we do, does it need to go on every single file? Because some files are missing it.

Reply:
Ah, I based this on one of the existing misuse handlers so if they need it this probably does as well. What's the convention for names/dates on these?
That said, I have no idea if we need it, though I don't recall any previous projects I worked on doing it per-file at least... I'd have thought the whole repo would be under one shared license document or something like that unless there's something about these files in specific...

Reply:
Should I remove it for the new handler or keep it in for consistency with the others at least for now?

Reply:
remove for the new one please